packages/blade/src/components/Box/BaseBox/types/propsTypes.ts

```diff
@@ -97,6 +97,10 @@ type GridProps = MakeObjectResponsive<
   >
 >;
 
+type PolygonSupportProps = MakeObjectResponsive<
+  PickCSSByPlatform<'transform' | 'transformOrigin' | 'clipPath'>
+>;
```

Comment:
Should we add this to `CommonBoxVisualProps`? Since that is where we have other equivalent props and `transform` and `transformOrigin` are not polygon-specific props.